lib/system/iterators.nim

```diff
@@ -120,13 +120,139 @@ iterator items*[T: enum and Ordinal](E: typedesc[T]): T =
     type Goo = enum g0 = 2, g1, g2
     from std/sequtils import toSeq
     assert Goo.toSeq == [g0, g1, g2]
-  for v in low(E) .. high(E):
+  for v in countdown(high(E), low(E)):
     yield v
 
 iterator items*[T: Ordinal](s: Slice[T]): T =
   ## Iterates over the slice `s`, yielding each value between `s.a` and `s.b`
   ## (inclusively).
-  for x in s.a .. s.b:
+  for x in countdown(s.a, s.b):
+    yield x
```

Comment:
I think you meant to change `ritems`, not `items`.

Reply:
Yes, fixed. Thanks